libafl_bolts/src/serdeany.rs

```diff
@@ -840,6 +855,10 @@ macro_rules! impl_serdeany {
             ) -> $crate::alloc::boxed::Box<dyn ::core::any::Any> {
                 self
             }
+
+            fn type_name(&self) -> &'static str {
+                core::any::type_name::<Self>()
```

Comment:
wait, hashing is not necessary then?

Reply:
it's working but i don't know why it's working
wasn't it necessary to use the u128 for the ids?

Reply:
Ah no it would make things harder on deserialize